terraform/modules/ecs/ecs.tf

```diff
@@ -309,6 +310,7 @@ data "template_file" "worker_container_definition" {
     domain                                       = "${var.domain}"
     audit_spreadsheet_id                         = "${var.audit_spreadsheet_id}"
     google_drive_json_key                        = "${replace(jsonencode(var.google_drive_json_key), "/([\"\\\\])/", "\\$1")}"
+    big_query_api_json_key                           = "${replace(jsonencode(var.big_query_api_json_key), "/([\"\\\\])/", "\\$1")}"
```

Comment:
I wonder whats this funky regex?

Reply:
Could be a nice thing to look into a Learn Tech 😄